Modix.Services/Moderation/ModerationService.cs

```diff
@@ -183,11 +152,8 @@ public class ModerationService : IModerationService
         }
```

Comment:
One of the main goals of splitting here is to decouple auth from any operations that aren't initiated from a frontend. So, `IAuthorizationService` should not be pulled in here. I think the only remaining uses here are to grab `CurrentUserId` and `CurrentGuildId` to record as `createdBy` or `deletedBy` or whatever. For these cases, these values should be passed in as parameters to the service method. So, `IModerationOperations` will just grab `IAuthorizationService.CurrentUserId` and `IAuthorizationService.CurrentGuildId` as needed, while auto-initiated stuff will just grab the `ISelfUser`.